linux/system.go

```diff
@@ -83,6 +84,25 @@ func (ls *linuxSystem) ScanDisks(filter disko.DiskFilter,
 	return disks, nil
 }
 
+func getDiskReadOnlyProperty(d disko.UdevInfo) (bool, error) {
+	syspath, err := getSysPathForBlockDevicePath(d.Name)
+	if err != nil {
+		return false, err
+	}
+
+	syspathReadOnly := syspath + "/ro"
+	if disko.PathExists(syspathReadOnly) {
```

Comment:
Are there linux disks that do not have this?

I think I'd rather just return error if /ro didn't exist.
here is a bit of shell test on my system:

```
$ for x in /sys/class/block/*; do read val < $x/ro || { echo "$x: no ro"; break; }; echo ${x##*/} $val; done
dm-0 0
dm-1 0
loop0 1
loop1 1
loop10 1
loop2 1
loop3 1
loop4 1
loop5 1
loop6 1
loop7 1
loop8 1
loop9 1
mmcblk0 0
mmcblk0boot0 1
mmcblk0boot1 1
mmcblk0p1 0
mmcblk0p2 0
mmcblk0p3 0
mmcblk0p4 0
sda 0
sda1 0
sda2 0
sda3 0
```

Reply:
AFAICT, it's only 0 ro 1.

https://elixir.bootlin.com/linux/latest/source/include/linux/genhd.h#L229 

> I think I'd rather just return error if /ro didn't exist. 

I can do that, but I'm confident we get either a 1 or a 0.